frontend/src/types.ts

```diff
@@ -4666,6 +4666,7 @@ export type HogFunctionInputSchemaType = {
     required?: boolean
     default?: any
     secret?: boolean
+    hidden?: boolean
```

Comment:
One thing - we have this in the plugin-server as well. It's not a biggy but would be cool to add it to the types there as well so we dont forget about it